hazelcast/src/main/java/com/hazelcast/cp/internal/CPMemberInfo.java

```diff
@@ -86,7 +87,11 @@ private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundE
         endpoint = new RaftEndpointImpl(uuid);
         String host = in.readUTF();
         int port = in.readInt();
-        address = new Address(host, port);
+        try {
+            address = new Address(host, port);
+        } catch (UnknownHostException ex) {
+            address = Address.createUnresolvedAddress(host, port);
```

Comment:
can you add a test for serialization/deserialization of `CPMemberInfo`?

Reply:
Yes, I will do that.